File: silver/11652/c++/main.cpp
#include <iostream>
#include <map>

int main()
{   

    std::map<long long, long long> arr;
    
    long long num, temp;
    std::cin >> num;

    for (int i = 0; i < num; i++) {
        std::cin >> temp;
        if (arr.find(temp) != arr.end()) {
            arr[temp]++;
        } else {
            arr[temp] = 1;
        }
    }
    long long  max = 0, index;
    for (const auto& [key, value] : arr) {
        if (max < value) {
            index = key;
            max = value;
        }
    }
    std::cout << index;
    return 0;
}

File: silver/11652/python/main.py

if (__name__ == "__main__"):
    num = int(input())

    dic = dict()

    for i in range(num):
        temp = int(input())
        if (temp in dic.keys()):
            dic[temp] += 1
        else:
            dic[temp] = 1

    max = 0; index = 0
    for key in sorted(dic.keys()):
        if (max < dic[key]):
            max = dic[key]
            index = key
    print(index)